glean-core/src/internal_metrics.rs

```diff
@@ -88,3 +88,26 @@ impl UploadMetrics {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct DatabaseMetrics {
+    pub size: MemoryDistributionMetric,
+}
+
+impl DatabaseMetrics {
+    pub fn new() -> DatabaseMetrics {
+        DatabaseMetrics {
+            size: MemoryDistributionMetric::new(
+                CommonMetricData {
+                    name: "size".into(),
+                    category: "glean.database".into(),
+                    send_in_pings: vec!["metrics".into()],
+                    lifetime: Lifetime::Application,
```

Comment:
This doesn't match the lifetime in the `metrics.yaml` (which defaults to ping).

Reply:
Yeah, as before forgot to update, but we need to figure out above questions before anyway